source/reporting/index.html.md.erb

```diff
@@ -0,0 +1,135 @@
+---
+title: Reporting 
+weight: 70
+---
+
+# Reporting 
+
+You can extract reporting information from GOV.UK Pay manually, or via the
+API.
+
+## Manual reporting
+
+You can sign in to your [GOV.UK Pay admin
+account](https://selfservice.payments.service.gov.uk/login) and export
+transaction information in a CSV file. 
+
+For a given account, select __Transactions__ in the __Dashboard__ page.
+
+You can filter transactions using the available search criteria, and then
+select __Download all transaction details as a CSV file__. 
+
+You can process CSV files using spreadsheet software.
```

Comment:
>  Isn't telling them to process CSV using spreadsheet software a bit patronising?

For some of the users, yes, but as I address elsewhere we're trying to meet the user needs of not-necessarily-technical product and service managers (who might not know about what a CSV is vis a vis spreadsheet software). 

> You could turn the last line into something like 'You now have your CSV report which you can process using spreadsheet software'. This is also a good content pattern as it reinforces the outcome and it also has a call to action.

I'm a little bit less sure of this suggestion because I think it might be unnecessarily long and I'm not sure if it needs a call to action in this particular case.